tests/test_type_view.py

```diff
@@ -257,20 +257,20 @@ def test_parsed_type_is_optional_predicate() -> None:
 
 def test_parsed_type_is_subclass_of() -> None:
```

Comment:
Sorry I missed this before, but the tests should probably be renamed as well.

Reply:
Thanks @guacs